client_windows_test.go

```diff
@@ -0,0 +1,43 @@
+// +build windows
+// Copyright 2016 go-dockerclient authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package docker
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"sync"
+
+	winio "github.com/Microsoft/go-winio"
```

Comment:
Nice catch @zramsay, thanks!

@samuelkarp can you remove the aliasing from this line and make this just "github.com/Microsoft/go-winio".